In [2]:
%load_ext autoreload
%autoreload 2

import numpy as np
import pandas as pd
import datetime
#graphing
import matplotlib.pyplot as plt

In [3]:
entry2_data = pd.read_csv("entry_loc2.csv")

## CDPs

### Total number of stores

In [4]:
# Total Lowe's
len(entry2_data[entry2_data['LOW']>0])

1192

In [5]:
# Total HD
len(entry2_data[entry2_data['HD']>0])

1255

There are total 1192 Lowe's and 1255 Home Depot stores. There are more HD stores.

### Single entry

In [6]:
# Lowe's with no HD presence
len(entry2_data[(entry2_data['LOW']>0) & (entry2_data['HD']==0)])

548

In [7]:
# HD with no Lowe's presence
len(entry2_data[(entry2_data['LOW']==0) & (entry2_data['HD']>0)])

611

Home Depot enters more markets with no Lowe's presence.

### Both Enter

In [8]:
# Number of markets Lowe's enters where HD has a presence
len(entry2_data[(entry2_data['LOW']>0) & (entry2_data['HD']>0)]) \
    - len(entry2_data[(entry2_data['LOW']>0) & (entry2_data['HD']==0)])

96

In [9]:
len(entry2_data[(entry2_data['LOW']>0) & (entry2_data['HD']>0)])

644

There are 644 common markets for both Lowe's and Home Depot. 

### Demographics

In [10]:
# Median income for Lowe's 
np.average(entry2_data[entry2_data['LOW']>0]['income_per_capita'])

29190.489932885906

In [11]:
# Median income for HD's markets
np.average(entry2_data[entry2_data['HD']>0]['income_per_capita'])

31595.29322709163

Home Depot enters markets with higher median income.

In [12]:
# Average population for Lowe's 
np.average(entry2_data[entry2_data['LOW']>0]['population'])

70177.18456375839

In [13]:
# Average population for HD's 
np.average(entry2_data[entry2_data['HD']>0]['population'])

82425.24143426295

Home Depot enters markets with higher population.

## CBSA

In [14]:
entry2_data_CBSA = entry2_data[entry2_data['population']>10000]

### Total number of stores

In [15]:
# Total Lowe's
len(entry2_data_CBSA[entry2_data_CBSA['LOW']>0])

972

In [16]:
# Total HD's
len(entry2_data_CBSA[entry2_data_CBSA['HD']>0])

1056

There are total 972 Lowe's and 1057 Home Depot stores. There are more HD stores.

### Single entry

In [17]:
len(entry2_data_CBSA[(entry2_data_CBSA['LOW']>0) & (entry2_data_CBSA['HD']==0)])

384

In [18]:
len(entry2_data_CBSA[(entry2_data_CBSA['LOW']==0) & (entry2_data_CBSA['HD']>0)])

468

Home Depot enters more markets with no Lowe's presence.

### Both Enter

In [19]:
# Number of markets Lowe's enters where HD has a presence
len(entry2_data_CBSA[(entry2_data_CBSA['LOW']>0) & (entry2_data_CBSA['HD']>0)]) \
    - len(entry2_data_CBSA[(entry2_data_CBSA['LOW']>0) & (entry2_data_CBSA['HD']==0)])

204

In [20]:
len(entry2_data_CBSA[(entry2_data_CBSA['LOW']>0) & (entry2_data_CBSA['HD']>0)])

588

There are 588 common markets for both Lowe's and Home Depot. 

### Demographics

In [63]:
# Median income for Lowe's 
np.average(entry2_data_CBSA[entry2_data_CBSA['LOW']>0]['income_per_capita'])

29462.10185185185

In [64]:
# Median income for HD's markets
np.average(entry2_data_CBSA[entry2_data_CBSA['HD']>0]['income_per_capita'])

31421.590350047303

Home Depot enters markets with higher median income.

In [65]:
# Average population for Lowe's 
np.average(entry2_data_CBSA[entry2_data_CBSA['LOW']>0]['population'])

84739.7767489712

In [66]:
# Average population for Lowe's 
np.average(entry2_data_CBSA[entry2_data_CBSA['HD']>0]['population'])

97108.2412488174

Home Depot enters markets with higher population.